heaven_besieged_ultimate.SC2Map/scripts/main.galaxy

```diff
@@ -0,0 +1,34 @@
+const int MAX_PLAYERS = 7;
```

Comment:
I take it these scripts are placeholders for latter additions?

Reply:
Yeap absolutely. These are copy/pastes from the guides, with no actual functionality ingame (excepting the noted duplicative chat command system). This is the noted 'smidgen of additional jank' laying out potential patterns. Those variable names were probably meant to be evocative of use cases, but being dropped into an existing map tbh also add confusion lol